From 827c9118b1de82044cf7a370b96a676a0b49a03f Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Wed, 9 Oct 2019 22:00:42 -0400
Subject: [PATCH] asus-flash: Remove hardcoded IDs, for USER_NS

Big oof... this section of code is not good.

This makes the build work woth USER_NS.
---
 .../msm/camera_v2/sensor/flash/msm_led_i2c_trigger.c      | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/media/platform/msm/camera_v2/sensor/flash/msm_led_i2c_trigger.c b/drivers/media/platform/msm/camera_v2/sensor/flash/msm_led_i2c_trigger.c
index 3593d29b79f..7c84a057b93 100644
--- a/drivers/media/platform/msm/camera_v2/sensor/flash/msm_led_i2c_trigger.c
+++ b/drivers/media/platform/msm/camera_v2/sensor/flash/msm_led_i2c_trigger.c
@@ -2247,14 +2247,10 @@ int msm_flash_i2c_probe(struct i2c_client *client,
 #endif
 	/*For ASUS FLASH+++*/
 	proc_entry_flash = proc_create_data("driver/asus_flash", 0666, NULL, &asus_flash_proc_fops, dummy);
-	proc_set_user(proc_entry_flash, 1000, 1000);
 	proc_entry_flash_second = proc_create_data("driver/asus_flash2", 0666, NULL, &asus_flash_second_proc_fops, dummy2);
-	proc_set_user(proc_entry_flash_second, 1000, 1000);
 	if( is_ZD550KL() ){
 		proc_entry_flash_third = proc_create_data("driver/asus_flash3", 0666, NULL, &asus_flash_third_proc_fops, dummy3);
-		proc_set_user(proc_entry_flash_third, 1000, 1000);
 		proc_entry_flash_fourth = proc_create_data("driver/asus_flash4", 0666, NULL, &asus_flash_fourth_proc_fops, dummy4);
-		proc_set_user(proc_entry_flash_fourth, 1000, 1000);
 	}
 	/*For ASUS FLASH---*/
 	pr_info("%s:%d probe success\n", __func__, __LINE__);
@@ -2349,14 +2345,10 @@ int msm_flash_probe(struct platform_device *pdev,
 
 	/*For ASUS FLASH+++*/
 	proc_entry_flash = proc_create_data("driver/asus_flash", 0666, NULL, &asus_flash_proc_fops, dummy);
-	proc_set_user(proc_entry_flash, 1000, 1000);
 	proc_entry_flash_second = proc_create_data("driver/asus_flash2", 0666, NULL, &asus_flash_second_proc_fops, dummy2);
-	proc_set_user(proc_entry_flash_second, 1000, 1000);
 	if( is_ZD550KL() ){
 		proc_entry_flash_third = proc_create_data("driver/asus_flash3", 0666, NULL, &asus_flash_third_proc_fops, dummy3);
-		proc_set_user(proc_entry_flash_third, 1000, 1000);
 		proc_entry_flash_fourth = proc_create_data("driver/asus_flash4", 0666, NULL, &asus_flash_fourth_proc_fops, dummy4);
-		proc_set_user(proc_entry_flash_fourth, 1000, 1000);
 	}
 	/*For ASUS FLASH---*/
 	mutex_init(&flash_lock);
-- 
2.23.0

